src/nunavut/lang/c/__init__.py

```diff
@@ -810,3 +810,16 @@ def filter_full_reference_name(language: Language, t: pydsdl.CompositeType) -> s
         return filter_id(language, not_stropped)
     else:
         return not_stropped
+
+@template_language_filter(__name__)
+def filter_to_standard_bit_length(language: Language, t: pydsdl.PrimitiveType):
+    """
+    Provides an int that returns the nearest valid standard bit length.
+    Added to support cleaner deserialization templating, there may be a
+    better way than this, unsure.
+
+    Not sure what to put here for testing, etc.
```

Comment:
I thought the code blocks in the above templates were for testing (a la Rust doc tests), but I guess they aren't.

Currently unsure what I need for documentation here.

Reply:
Yes. `tox -e doctest` executes the doctests. Search for `.. code-block: python` and `.. invisible-code-block: python` for examples in the src.

Reply:
FYI: there is a test fixture provided to cover jinja filters. See https://github.com/UAVCAN/nunavut/blob/master/src/nunavut/lang/c/__init__.py#L109 for an example doctest and https://github.com/UAVCAN/nunavut/blob/master/conftest.py#L245 for the filter itself.